src/pages/signin/PasswordForm.js

```diff
@@ -55,7 +55,7 @@ class PasswordForm extends React.Component {
     }
 
     componentDidMount() {
-        if (!this.input) {
+        if (!this.input || !this.props.isVisible) {
```

Comment:
Any reason not to use `!canFocusInputOnScreenFocus()` over here?
Not using it causes this jarring keyboard bug when you launch the app in the password screen.

https://user-images.githubusercontent.com/29673073/148661082-9ab4bf65-07cd-4a36-879a-aab1344a111c.mp4

Reply:
Ah I see, you had asked about this in the issue.
> The !canFocusInputOnScreenFocus() is only present on the login form and prevents the autofocus on touch screen devices. It seems counterintuitive to the current solution.

I still think we should keep it in both screens. Because once you open the keyboard on login screen, the keyboard stays open on password screen too. And it avoids the keyboard bug.

Thoughts?

Reply:
@rushatgabhane it was never present on the password form but I agree, it should be. Not to make this more complex then it needs to be but the whole intent of the didMount function is autofocus, and  it is now being excluded from the most common use cases (all of mobile). That bug is only present on android, and could be addressed on android with platform specific code. I'll edit the current pr and wait for your feedback.

Reply:
Hmm true. Then the question is do we want to autofocus on login page for mobile? Because the current user flow is good imo.

If yes, we'll create a new issue for it.
I'm gonna start a discussion on it on slack. Feel free to join the community if you haven't already and give your input.

Reply:
Generally agreed. I sent two emails to join slack this past week and haven't gotten an invite yet.